vitrina/datasets/views.py

```diff
@@ -175,3 +178,20 @@ def get_context_data(self, **kwargs):
         }
         context_data.update(extra_context_data)
         return context_data
+
+
+class DatasetMembersView(LoginRequiredMixin, UserPassesTestMixin, ListView):
+    login_url = '/login/'
+    model = DatasetMember
+    template_name = 'vitrina/datasets/members_list.html'
+    context_object_name = 'dataset_members'
+    paginate_by = 20
+
+    def test_func(self):
+        dataset = Dataset.public.get_from_url_args(**self.kwargs)
+        return Representative.objects.filter(organization_id=dataset.organization.id,
+                                             email=self.request.user.email).exists()
```

Comment:
Use [PermissionRequiredMixin.has_permission](https://docs.djangoproject.com/en/3.2/topics/auth/default/#django.contrib.auth.mixins.PermissionRequiredMixin.has_permission) instead.

`UserPassesTestMixing` is for checking if a correct user has been logged in, and `PermissionRequiredMixin` is for simple permission checks.